README.md

```diff
@@ -200,6 +200,53 @@ KUBECOLOR_OBJ_FRESH="1m" kubecolor get po
 
 Default value is `0s`, it means is disabled.
 
+### Change Color Schema
+
+As of version `v0.3.0` it is now possible to use your own color scheme !
+
+To do so, define the environment variable `KUBECOLOR_CUSTOM_COLOR` as a SemiColums (`;`) delimited string, setting one or many of the available keys:
+
+Key      | Description
+---      | --- 
+default  | used when no specific mapping is found for the command
+key      | used to color keys in a `key:value` situation ; value will be either on the following types
+string   | used for single `strings`
+true     |  used when value is `true`
+false    | used when value is `false` or an `error`
+number   | used when the value is a `number`
+null     | used when the value is `null` or a `warning`
+header   | used to print top headers
+fresh    | used for a `delay` or `duration`, like a pod's age
+required |  used when the value is required or is an `error`
+random   | list of comma separated numbers, used to cycle color of each column
+
+
+Use possible color / values from this table (colors may vary depending on your terminal settings, this color chart is just a reference and results may differ)
+
+Color                                           | Value
+---                                             | ---
+Black                                           | 30
+<font color="red">Red</font>                    | 31
+<font color="green">Green</font>                | 32
+<font color="yellow">Yellow</font>              | 33
+<font color="blue">Blue</font>                  | 34
+<font color="magenta">Magenta</font>            | 35
+<font color="cyan">Cyan</font>                  | 36
+<font color="white">White</font>                | 37
+<font color="grey">Light Black ( Grey)</font>   | 90
+<font color="orange">Light Red</font>           | 91
+<font color="lime">Light Green</font>           | 92
+<font color="yellow">Light Yellow</font>        | 93
+<font color="lightblue">Light Blue</font>       | 94
+<font color="pink">Light Magenta</font>         | 95
+<font color="aquamarine">Light Cyan</font>      | 96
+<font color="silver">Light White (White)</font> | 97
```

Comment:
I think allowing more than just the basic ANSI coloring set would be nice. Like what if we use an off the shelf library instead?

There's the tried and tested [faith/color](https://github.com/fatih/color), but the new kid on the block is [gookit/color](https://github.com/gookit/color), which has [its own color tag syntax](https://pkg.go.dev/github.com/gookit/color#ParseCodeFromAttr) (e.g `fg=yellow,bg=blue,op=bild,underscore`). It also, if I understand it correctly, converts colors automatically in case your terminal only supports say 16 or 256 colors.

So we could turn this environment variable into multiple environment variables, like so:

```properties
KUBECOLOR_THEME_DEFAULT=""
KUBECOLOR_THEME_NULL="fg=gray"
# etc
```

Reply:
I will check that out... for sure my "simple" solution is just changing the color schema and not much the code itself... maybe a first step ? I'll take more time to review all that during end of year.

Reply:
Having a simple first step is good, as long as it's open to extension.

For example, if we had per-color scheme key environment variables, then the initial implementation could just be "must be a fixed integer/color name", so we could extend it later with "if fail to parse as int, then try parse with the colorable library's parse function"

But if we have all color scheme keys merged into one with our own parsing, then it'll be much trickier to evolve that solution without replacing it.